Numerics/src/Numerics.csproj

```
@@ -4,12 +4,13 @@
     <TargetFramework>netstandard2.1</TargetFramework>
     <AssemblyName>Microsoft.Quantum.Numerics</AssemblyName>
     <QsharpDocsGeneration>true</QsharpDocsGeneration>
+    <QsharpDocsPackageId>Microsoft.Quantum.Numerics</QsharpDocsPackageId>
     <IncludeQsharpCorePackages>false</IncludeQsharpCorePackages>
   </PropertyGroup>
 
   <PropertyGroup>
     <NoWarn>0162</NoWarn>
-    <RunQDocGen>True</RunQDocGen>
+    <RunQDocGen>True</RunQDocGen>https://github.com/microsoft/qsharp-runtime/pull/390
```

That does not look like valid XML

---

It's not at all, no. I'll blame a copy-paste error. Thanks for catching it!
```suggestion
    <RunQDocGen>True</RunQDocGen>
```